src/model/image-tag.js

```diff
@@ -34,7 +34,7 @@ class ImageTag {
   }
 
   toString() {
-    return `${this.image}:${this.tag}`;
+    return `${this.image}:${this.tag}-base-0`;
```

Comment:
This does not seem correct. ToString should not magically show something else than the value of the object.

Reply:
this is not test code, repo version and platform will need to be hardcoded to pull the correct editor image if we add a way to get repo version that can also become a property of image-tag, `base` will still need to be hard coded as there is no tag on docker hub without a platform

I can move this hardcoding somewhere else also but the issue for test will still be the same, since the test builds an alpine image we don't want to pass `-base-0` in the input base image as an arg to docker build command

Reply:
Ah right I misunderstood indeed. LGTM